### PR TITLE
SWI-3927 OpenAPI Generator SDK

### PR DESCRIPTION
**Pull request by ajrice6713**

### CODE REVIEW

File: bandwidth/models/bxml/verbs/forward.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                away
+
+                This parameter is considered only when diversionTreatment is set to stack. Defaults is unknown. 
+                Defaults to None.
+            uui (str, optional): The value of the User-To-User header to send within the outbound INVITE when forwarding to a SIP URI. 
+                Must include the encoding parameter as specified in RFC 7433. Only base64 and jwt encoding are currently allowed. 
+                This value, including the encoding specifier, may not exceed 256 characters.
+        """
+        self.to = to
+        self._from = _from
```
Comment (juliajanu) [inline bandwidth/models/bxml/verbs/forward.py:50]: I want to say that out of all the verbs and all the verb attributes this is the only one prepended with an underscore. Why? If there is an actual reason then I guess my only comment would be that the comment above says:
`from_ (str, optional): ...` which wouldn't match

File: test/unit/bxml/test_redirect.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+from bandwidth.models.bxml import Verb
+from bandwidth.models.bxml import Redirect
+
+
+class TestRedirect(unittest.TestCase):
+
+    def setUp(self):
+        self.redirect = Redirect("https://example.com/redirect")
+        # self.redirect.redirect_url = "https://example.com/redirect"
```
Comment (juliajanu) [inline test/unit/bxml/test_redirect.py:18]:
```suggestion
```

File: bandwidth/models/bxml/verbs/forward.py (hunk 1 above)
Comment (ajrice6713) [reply bandwidth/models/bxml/verbs/forward.py:50]: `from` is a reserved word in python, so it cant be used as a property name

Comment (ajrice6713) [reply bandwidth/models/bxml/verbs/forward.py:50]: Will fix the comment 😬

**Review by juliajanu — APPROVED** [review]

**Review by brianluisgomez — APPROVED** [review]

**Pull request merged**